router/sources/router.move

```diff
@@ -0,0 +1,928 @@
+module router::router {
```

Comment:
Can you add more documentations on when these different functions are used? There's some overlapping in a few of them such as migrate and renew